pkgs/applications/audio/bitwig-studio/bitwig-studio5.nix

```diff
@@ -27,11 +27,12 @@
 
 stdenv.mkDerivation rec {
   pname = "bitwig-studio";
-  version = "5.1.8";
+  version = "5.1.9";
 
   src = fetchurl {
-    url = "https://downloads.bitwig.com/stable/${version}/${pname}-${version}.deb";
-    sha256 = "sha256-KxNLae/uTYL1m/X+/7wr7hhKfw31NpB9Mw9RzfrTuus=";
+    url = "https://www.bitwig.com/dl/Bitwig%20Studio/${version}/installer_linux/";
```

Comment:
How about using the canonical `https://downloads-secure.bitwig.com/5.1.9/bitwig-studio-5.1.9.deb` ?